cvat/apps/organizations/templates/invitation/invitation_message.html

```diff
@@ -122,14 +122,9 @@
         <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px;">
           <tr>
             <td align="center" valign="bottom">
-              <a href="https://www.cvat.ai/" target="_blank" style="display: inline-block;">
-                <img
-                  src="https://github.com/cvat-ai/cvat/blob/develop/site/content/en/images/cvat_poster_with_name.png?raw=true"
-                  alt="Logo"
-                  border="0"
-                  width="100%"
-                  style="display: block; width: 100%;"
-                >
+              <a href="https://www.cvat.ai/" target="_blank">
+                <img src="https://cdn.prod.website-files.com/62bebfb83044de7f13ae257e/62bec02423796903b293473e_Logo.svg"
```

Comment:
May we use the icon from a release, instead of 3rdparty cdn? 

E.g. `https://raw.githubusercontent.com/cvat-ai/cvat/v2.18.0/cvat-ui/src/assets/cvat-logo.svg`

Reply:
done. Thanks.